src/main/java/io/github/aquerr/steamwebapiclient/response/PlayerGetRecentlyPlayedGamesResponse.java

```diff
@@ -0,0 +1,44 @@
+package io.github.aquerr.steamwebapiclient.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Class representing the result of <a href="https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames">https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames</a>
+ */
+@Data
+public class PlayerGetRecentlyPlayedGamesResponse implements SteamWebApiResponse {
+
+    @JsonProperty("response")
+    private GetRecentlyPlayedGamesResponse response;
+
+    @Data
+    public static class GetRecentlyPlayedGamesResponse {
+        @JsonProperty("total_count")
+        private int totalCount;
+        @JsonProperty("games")
+        private List<GameDetails> gameList;
+
+        @Data
+        public static class GameDetails {
+            @JsonProperty("appid")
+            private int appId;
+            @JsonProperty("name")
+            private String name;
+            @JsonProperty("playtime_2weeks")
+            private int playtime2Weeks;
+            @JsonProperty("playtime_forever")
+            private int playtimeForever;
```

Comment:
I am wondering if we should change those fields type to long maybe? 🤔
Also, would be great if we could tell if those are seconds, minutes or hours. Maybe calling the field "playtimeForeverMinutes" would be enough.